File: hello_world.py
def hello_world(name: str = "World") -> str:
    """
    Simple Hello World function in Python

    Args:
        name: Optional name to greet (defaults to "World")

    Returns:
        Greeting message
    """
    return f"Hello, {name}!"

# Example usage
if __name__ == "__main__":
    print(hello_world())  # "Hello, World!"
    print(hello_world("AgentForge"))  # "Hello, AgentForge!"

File: hello-world.js
/**
 * A simple hello world function that demonstrates basic JavaScript syntax
 * and console output.
 *
 * @param {string} [name="World"] - The name to greet
 * @returns {string} The greeting message
 */
function helloWorld(name = "World") {
    const greeting = `Hello, ${name}!`;
    console.log(greeting);
    return greeting;
}

// Example usage
helloWorld(); // Outputs: "Hello, World!"
helloWorld("Claude"); // Outputs: "Hello, Claude!"
helloWorld("AgentForge"); // Outputs: "Hello, AgentForge!"

// Export for use in other modules
module.exports = helloWorld;